src/cartridges/int_adyen_overlay/cartridge/scripts/adyenZeroAuth.js

```diff
@@ -36,12 +37,19 @@ function zeroAuthPayment(customer, paymentInstrument) {
     if (AdyenHelper.getAdyen3DS2Enabled()) {
       zeroAuthRequest = AdyenHelper.add3DS2Data(zeroAuthRequest);
     }
+
     zeroAuthRequest.amount = {
       currency: session.currency.currencyCode,
       value: 0,
     };
 
+    zeroAuthRequest.returnUrl = URLUtils.https('Adyen-Redirect3DS1Response').toString();
+
+    // StorepaymentMethod overrides enableReccuring/enableOneClick. Either one is allowed, not all.
     zeroAuthRequest.storePaymentMethod = true;
+    delete zeroAuthRequest.enableRecurring;
```

Comment:
Let's delete that in the adyenHelper already where we add it in the zeroAuthRequest

Reply:
Discussed this over Mattermost. This wil be resolved in PW-2457